docker/Dockerfile.debug

```diff
@@ -11,10 +11,11 @@ RUN apt-get update && \
     net-tools \
     libc6-dbg gdb valgrind \
     vim \
-    nano \
-    emacs \
-    wrk
```

Comment:
we should have a separate "dev/testing env" image than one for proxy_debug
it seems quite wasteful to have 3 editors and a load tester but for the sake of getting the PR in I guess I'll "add" only